app/admin/user.rb

```diff
@@ -56,7 +56,7 @@
   end
 
   action_item :user_registration, only: :show, priority: 0 do
-    link_to I18n.t('resend_user_registration'), resend_user_registration_admin_user_path, method: :patch
+    link_to I18n.t('resend_user_registration'), resend_user_registration_admin_user_path, method: :patch unless user.confirmed?
```

Comment:
instead of using `unless` in here you can pass it as an option on `action_item` instead, check this example on `admin/questionnaire.rb`

https://github.com/Codeminer42/Punchclock/blob/21341156199ea8a8e78da25c85f5232888ee9f56/app/admin/questionnaires.rb#L18-L21